### PR TITLE
Remove TODO's from nl 2020 markup

### PR DESCRIPTION
**Pull request by noah-vdv**
Makes progress on #1750

### CODE REVIEW

File: src/content/nl/2020/markup.md [hunk 1]
```diff
@@ -177,16 +171,13 @@ In totaal bevat ongeveer 2% van de pagina's helemaal geen scripts, zelfs niet me
 
 Aan de andere kant van het spectrum laten de cijfers zien dat ongeveer 97% van de pagina's ten minste één script bevat, inline of extern.
 
-{# TODO(analysts): We still have a problem here with the x-axis label (“Containing”). Can someone help out and look at this? #}
-
 {{ figure_markup(
   image="script-use.png",
+  alt="Gebruik van het script element
```
Comment (bazzadp) [inline src/content/nl/2020/markup.md:176]:
```suggestion
  alt="Gebruik van het script element,
```

**Review by bazzadp — CHANGES_REQUESTED** [review]
One change to fix the build.

Also did you see the changes in https://github.com/HTTPArchive/almanac.httparchive.org/pull/1858/files?

File: src/content/nl/2020/markup.md (hunk 1 above)
Comment (noah-vdv) [reply src/content/nl/2020/markup.md]: Ah yes I'll add those

**Comment by noah-vdv** [comment]
> One change to fix the build.
> 
> Also did you see the changes in https://github.com/HTTPArchive/almanac.httparchive.org/pull/1858/files?

Ah yes I'll add those

File: src/content/nl/2020/markup.md (hunk 1 above)
Comment (noah-vdv) [reply src/content/nl/2020/markup.md]: And it also needs the " after element right?

**Comment by noah-vdv** [comment]
It also needs `"` after "script element" right?

File: src/content/nl/2020/markup.md [hunk 2]
```diff
@@ -177,16 +171,13 @@ In totaal bevat ongeveer 2% van de pagina's helemaal geen scripts, zelfs niet me
 
 Aan de andere kant van het spectrum laten de cijfers zien dat ongeveer 97% van de pagina's ten minste één script bevat, inline of extern.
 
-{# TODO(analysts): We still have a problem here with the x-axis label (“Containing”). Can someone help out and look at this? #}
-
 {{ figure_markup(
   image="script-use.png",
+  alt="Gebruik van het script element,
```
Comment (bazzadp) [inline src/content/nl/2020/markup.md:176]: Well a `"` - that explains why my last one didn't fix the build!

```suggestion
  alt="Gebruik van het script element",
```

**Review by bazzadp — APPROVED** [review]

**Comment by bazzadp** [comment]
Thank you! Hopefully other chapters won’t have as many changes after translations!

**Pull request merged**